CHANGELOG-LATEST.md

```diff
@@ -1,5 +1,33 @@
-### Dependency Updates
-* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 8.1.1 (#892) via RevenueCat Git Bot (@RCGitBot)
-  * [Android 7.2.7](https://github.com/RevenueCat/purchases-android/releases/tag/7.2.7)
-  * [iOS 4.31.2](https://github.com/RevenueCat/purchases-ios/releases/tag/4.31.2)
-* Bump danger from 9.4.0 to 9.4.1 (#889) via dependabot[bot] (@dependabot[bot])
+### New Features
+*   📱 Initial support for cross-platform RevenueCat Paywalls 🐾 🧱  (#852) 
+
+#### Instructions:
+- Update `purchases-flutter` in your `pubspec.yaml`:
+```yaml
+dependencies:
+  purchases_flutter: 6.6.0-beta.3
+```
+- For Android, you need to change your `MainActivity` to subclass `FlutterFragmentActivity` instead of `FlutterActivity`.
```

Comment:
I am worried folks stop reading after the first step of adding the yaml, should we put this first thing?

Reply:
Hmm yeah I think that makes sense!